TESTING.md

```diff
@@ -24,15 +24,13 @@ Tests should be fast where possible.
 
 
 ## Testing Pyramid
-Job Server uses a three layer testing pyramid, with the following layers from top to bottom:
-* End-to-End
-* Integration
-* Unit
 
-Below we describe how we draw the lines between each of those layers.
+Job Server uses a testing pyramid, with the following layers from top to bottom:
 
-Much like the developer tooling at Bennett Institute for Applied Data Science we define these layers as an ideal guide that should be adhered by default, but in the knowledge that things can deviate if there is a good reason.
+* Integration
+* Unit
 
+**At present, there are no end-to-end tests.**
```

Comment:
This line isn't necessary. We don't mention there aren't any other types of test, so why mention that there aren't any end-to-end tests?

Reply:
I left it in because it's part of the typical pyramid testing, but it's fine to take the line now, so let's do that.